common/src/stack/switch/pylib/switch/m7800.py

```diff
@@ -72,7 +72,12 @@ def connect(self):
 
 		self.proc.conversation(login_seq)
 
-		self._api_connection = mellanoknok.Mellanoknok(self.switch_ip_address, password=self.password)
+		# don't worry if we can't connect -- too many connection attempts breaks the api
+		# the library will auto try again if there's a method that actually needs it
+		try:
+			self._api_connection = mellanoknok.Mellanoknok(self.switch_ip_address, password=self.password)
+		except:
```

Comment:
Should this catch a specific exception instead of swallowing all potential exceptions?